recipe/gen_patch_json.py

```diff
@@ -994,13 +994,18 @@ def _gen_new_index(repodata, subdir):
             new_depends = record.get("depends", [])
             new_depends.append("libgcc-ng <=9.3.0")
             record["depends"] = new_depends
-        
+
         # setuptools started raising a warning when using `LooseVersion` from distutils
         # since packages don't tend to pin setuptools, this raises warnings in old versions
         # https://github.com/conda-forge/conda-forge.github.io/issues/1575
+        # This is what the time-based pin is for.
+        # NumPy and SciPy do not support setuptools >= 60.0, and perhaps never
+        # will, so they get a permanent pin. Other packages using
```

Comment:
Permanent pins should go in the recipe, instead of repodata patching.

Reply:
Okay thanks. That wasn't clear to me from the discussion in gh-204. Closing, will resubmit on the individual feedstocks.

Reply:
Sorry, I should have made it clear. We should patch the older packages here with a timestamp check.